docs/user/README.md

```diff
@@ -1,14 +1,43 @@
-> **TIP:** Apart from the {Module Name} heading, you can use your own titles for the remaining sections. You can also add more module-specific sections.
-
-# {Module Name}
-> Modify the title and insert the name of your module. Use Heading 1 (H1).
+# KIM Snatch
 
 ## Overview
-> Provide a description of your module and its components. Describe its features and functionalities. Mention the scope and add information on the CustomResourceDefinitions (CRDs).
-> You can divide this section to the relevant subsections.
+The KIM Snatch is part of Kyma Infrastructure Manager's (KIM) worker pool feature. It is deployed on all Kyma-managed runtimes. Mandatory modules are not visible for  SAP BTP, Kyma runtime customers and automatically installed by the [KLM](https://github.com/kyma-project/lifecycle-manager) on each SAP BTP, Kyma runtime.
```

Comment:
We might skip the sentence about mandatory modules now that the word "module" is not used in the name, and have one of the following instead: